### ISSUE TITLE
multiple image upload

### ISSUE THREAD

**Issue by truthSeekr** [locomotivecms/engine#1083]
How can I achive multiple image/file upload ? I have "Gallery" entry that has many "Photo" entries, but I can only add one photo...

**Comment by did** [comment]
@truthSeekr Does your photo content type have fields other than the file one? I'm asking you that because, perhaps, we just need a "gallery" field type.

**Comment by truthSeekr** [comment]
Yes my photo content type has fields other then file. It looks like this.
Photo content_type
    fields: 
    - desc: # Name of the field
        label: Opis
        type: string
        required: false
        hint: widoczny przy powiększeniu zdjęcia
        localized: false

    - image: # Name of the field
        label: Plik (zdjecie)
        type: file
        required: true
        hint: "format *.jpg, *.png"
        localized: false

- gallery:
    label: Galeria
    type: belongs_to
    required: true
    localized: false
    class_name: galleries

and Gallery content type is like this:
    fields: 
    - name: # Name of the field
        label: Nazwa
        type: string
        required: true
        hint: 
        localized: false

    - place: # Name of the field
        label: Miejsce
        type: string
        required: false
        hint: opcjonalne
        localized: false

    - data: # Name of the field
        label: Data
        type: date
        required: false
        hint: opcjonalne
        localized: false

    - photos:
        label: Zdjecia
        type: has_many
        required: false
        localized: false
        class_name: photos
        inverse_of: gallery
        ui_enabled: true

I add photos through gallery entry. So if the photo wouldnt have other fields than file I could upload multiple files at once ?

**Comment by did** [comment]
hey @truthSeekr! If the description is not important to you, yes, the 'gallery' field type would make totally sense here. 
We also could imagine something more powerful. We could tell a content type with a file field to act like a gallery. This way, that would enable some new functionalities like multiple uploads, display the content entries as a image grid....
